hyperspy/signal.py

```diff
@@ -2850,7 +2852,7 @@ def __repr__(self):
 
         return string.encode('utf8')
 
-    def __getitem__(self, slices, isNavigation=None):
+    def __getitem__(self, slices, isNavigation=None, out=None):
```

Comment:
Why not add this to the new slicing code (`hyperspy/misc/slicing.py`)? I though this will be removed with 0.9 anyway